### ISSUE TITLE
List of collections in repository lacks list-type structure element

### ISSUE THREAD

**Issue by alexdryden** [UIUCLibrary/arclight#21]
See: https://github.com/UIUCLibrary/arcuit/pull/7

Repository show (list of collections) – Works well with keyboard navigation, but the screen reader doesn’t announce it as a list, so users can’t know how many items are in the collection. Wrapping the items in a list structure would help with that:
```
<ul>
<li>
<div class="col-md-12">
   <div class="al-document-title-bar">
     <div class="row">
       <div class="col-md-10 col-lg-9">
           <h3 class="h5"><a href="/catalog/ALA-26-3-14">The Public-Access Computer Systems Review, 1990-1993</a></h3>
      </div>
        <div class="col text-right al-collection-id">
          Collection ID: ALA.26.3.14
        </div>
    </div>
  </div>
</div>
</li>
….
</ul>
```

**Comment by alexdryden** [comment]
@alexdryden TODO: this issue should be pushed upstream
